src/controllers/routes.ts

```diff
@@ -1,11 +1,13 @@
 import { Router } from '@well-known-components/http-server'
 import { GlobalContext } from '../types'
 import { resizeHandler } from './handlers/resize-handler'
+import { wellKnownComponents } from 'decentraland-crypto-middleware'
 
 // We return the entire router because it will be easier to test than a whole server
 export async function setupRouter(globalContext: GlobalContext): Promise<Router<GlobalContext>> {
   const router = new Router<GlobalContext>()
 
+  router.use(wellKnownComponents({}))
```

Comment:
I think you need to use this middleware but only on the same path (otherwise all endpoints would become signed-fetch).

Reply:
Yeah, I'm aware of that, but since this is the only endpoint I left it like that, anyways, I'll update to be scoped only for this endpoint to avoid future headaches :)